src/main/java/com/graphhopper/converter/core/Converter.java

```diff
@@ -3,13 +3,72 @@
 import com.graphhopper.converter.api.*;
 
 import javax.ws.rs.core.Response;
+
 import java.util.HashMap;
 import java.util.List;
 
 /**
- * @author Robin Boldt
+ * @author Robin Boldt, David Masclet
  */
 public class Converter {
+	
+	public static GHEntry convertFromGisgraphyAddress(GisgraphyAddressEntry gisgraphyEntry) {
+		GHEntry rsp = new GHEntry(null, null, gisgraphyEntry.getLat(),
+				gisgraphyEntry.getLng(), gisgraphyEntry.getDisplayName(), null,
+				gisgraphyEntry.getCountry(), gisgraphyEntry.getCity(),
+				gisgraphyEntry.getState(), gisgraphyEntry.getStreetName(),
+				gisgraphyEntry.getHouseNumber(), gisgraphyEntry.getZipCode());
+		return rsp;
+	}
+	
+	public static GHEntry convertFromGisgraphySearch(GisgraphySearchEntry gisgraphyEntry) {
+		GHEntry rsp = new GHEntry(null, null, gisgraphyEntry.getLat(),
+				gisgraphyEntry.getLng(), gisgraphyEntry.getLabel(), null,
+				gisgraphyEntry.getCountry(), gisgraphyEntry.getIsIn(),
+				gisgraphyEntry.getAdm1Name(), gisgraphyEntry.getName(),
+				gisgraphyEntry.getHouseNumber(), gisgraphyEntry.getZipCode());
+		return rsp;
+	}
+
+	public static Response convertFromGisgraphyList(
+			List<GisgraphyAddressEntry> gisgraphyEntries, Status status) {
+		if (gisgraphyEntries == null) {
```

Comment:
Have you seen cases where gisgraphyEntries were null? If this can happen we need a check for the other convert methods as well I think. I can do this in a separate PR.

Reply:
I always check input entry whatever the use, to avoid null pointer.